docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -34,6 +34,9 @@ mappings should be {ecs-ref}[ECS-compliant].
 * *{ml-cap} rules*: Creates an alert when a {ml} job discovers an anomaly above
 the defined threshold (see <<machine-learning>>).
 * *New terms*: Generates an alert for each new term detected in source documents within a specified time range.
+* *ES|QL*: Uses the Elasticsearch Query Language (ES|QL) rule to find events and aggregate search results.
```

Comment:
agree on moving both new terms and ESQL up

Your suggestion uses `uses` twice in one sentence. Let's rephrase it

Reply:
We can actually remove the first part that describes the rule type being used:

```suggestion
* *ES|QL*: Uses {ref}/esql.html[ES|QL] to find events and aggregate search results. 
```

Reply:
```suggestion
* *ES|QL*: Uses {ref}/esql.html[Elasticsearch Query Language (ES|QL)] to find events and aggregate search results.
```

I think it would be beneficial, to decode ESQL, similarly like in EQL example